bindings/nodejs/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "nodejs",
+  "version": "0.0.0",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "napi": {
+    "name": "nodejs",
+    "triples": {
+      "additional": [
+        "aarch64-apple-darwin",
```

Comment:
Is this the place where we control the target arches? Should probably be ["aarch64-apple-darwin", "x86_64-apple-darwin", "x86_64-unknown-linux-gnu", "x86_64-pc-windows-msvc"]

Reply:
no idea if this is the best place to control this. It was automatically added when I created the folder with `napi new`

Reply:
Either way I don't think we should support all these platforms arbitrarily since we can't provide guarantees that they work, and we would prefer developers to use the Rust library directly.